arrows/vxl/convert_image.cxx

```diff
@@ -320,7 +331,7 @@ ::apply_transforms( vil_image_view_base_sptr& view )
 template < typename opix_t, typename ipix_t >
 vil_image_view< opix_t >
 convert_image::priv
-::scale_and_convert( vil_image_view< ipix_t > input )
+::scale_and_convert( vil_image_view< ipix_t >& input )
```

Comment:
Uh... did you intend to make this a mutable reference? Maybe you meant `const&`?